### PR TITLE
Feature/food list

### PR DESCRIPTION
**Pull request by rachaouldbabaali**
in this Pull request, I have done the following :

- Food list :
    - [ ] Should display a list of food added by the logged-in user as in the wireframe (display also quantity of a given food).
    - [x] Should lead to a form that allows users to add new food.
    - [x] Make sure there are no N+1 queries happening.

- [x] Write unit and integration tests

### CODE REVIEW

File: app/views/foods/_form.html.erb [hunk 1]
```diff
@@ -11,32 +11,32 @@
...
+  <div class="mb-3">
     <%= form.label :quantity, style: "display: block" %>
     <%= form.number_field :quantity %>
   </div>
 
-  <div>
+  <div class="mb-3">
     <%= form.label :owner_id, style: "display: block" %>
     <%= form.text_field :owner_id %>
   </div>
```
Comment (andrianarivo) [inline app/views/foods/_form.html.erb]: You can remove this field because you don't use it on #create

File: app/views/foods/index.html.erb [hunk 2]
```diff
@@ -1,13 +1,10 @@
...
+<h1 class="mx-3">Foods</h1>
 
 <div id="foods">
   <% @foods.each do |food| %>
     <%= render food %>
-    <p>
-      <%= link_to "Show this food", food %>
-    </p>
   <% end %>
 </div>
```
Comment (andrianarivo) [inline app/views/foods/index.html.erb]: Add a new food link should be at **top right** corner of the page

**Review by andrianarivo — CHANGES_REQUESTED** [review]
Hi @rachaouldbabaali  , 

Good job so far!
There are some issues that you still need to work on to go to the next task but you are almost there!


## Highlights
✔️ Great commit messages
✔️ PRO PR
✔️ No linter error 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


 Happy coding!👏👏👏


------

**Comment by rachaouldbabaali** [comment]
Thank you @andrianarivo , I'll do the necessary work and I'll notify you

**Review by andrianarivo — APPROVED** [review]
## Hi @rachaouldbabaali  , 

The project setup is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

Image

Happy coding!👏👏👏


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rachaouldbabaali** [comment]
Thank you @andrianarivo i truly appreciate your time and efforts in reviewing my work 😃

**Pull request merged**